LayoutTests/svg/repaint/resources/animate-center.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="200" height="200">
+ <defs>
+ <style type="text/css"><![CDATA[
+       #square {
+         fill: red;
+         animation: keyframes 0.20s;
+         animation-fill-mode: forwards;
+         animation-timing-function: step-end;
+       }
+       @keyframes keyframes {
+         0% { fill: pink }
+         100% { fill: green }
+       }
```

Comment:
If you use the same value from `from` and `to`, and provided it's different to the underlying value (not `red`), does this help with the flakiness? You'd also need to ensure that this modified version of the test also fails prior to your patch.